### ISSUE TITLE
Cheetah hardcodes timezone

### ISSUE THREAD

**Issue by FilipeMaia** [antonbarty/cheetah-2017#17]
Cheetah hardcodes the timezone to US/Pacific. This can potentially cause problems when processing data from SACLA or other places.

**Comment by antonbarty** [comment]
Yes, this is old history.  We needed it at the time to make the timestamps match the log book (and not spit out Europe time)

We don’t want the user to have to configure everything from scratch, even though that’s the most flexible.
One way to solve this is to have a few standard setups: for example, experiment location = LCLS/CXI sets by default the cspad, and US/Pacific timezone etc.  experiment location = SACLA would set other parameters.

A.

**Comment by FilipeMaia** [comment]
A solution that might work automagically is to hardcode the timezone in the light source specific code part (cheetah_ana_mod and cheetah-sacla).

**Comment by antonbarty** [comment]
Yes, that would work. CXI is unlikely to suddenly appear in Japan or Germany. 

## Anton

Anton Barty
Centre for Free Electron Laser Science
+49 (0)170 639 0539
